01week/resume/resume.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="resumesheet.css" type="text/css">
+    <title>Cameron Harmon ~ Resume</title>
+</head>
+
+<body>
+    <nav>
+        <a href="home.html">Home</a> |
+        <a href="bio.html">Bio</a> |
+        <a href="resume.html">Resume</a> |
+        <a href="contact.html">Contact</a>
+    </nav>
+    <header>
+        <h1 id="name">Cameron Harmon</h1>
+        <p>Web Developer from Austin, TX looking for work.</p>
+    </header>
+
+    <main>
+        <section id="About">
```

Comment:
Since the section here only holds the p within it, you can take out the section tags and change p to read<p id="About">.